modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -233,7 +210,7 @@ The `identityProviders` stanza in the
 [discrete]
 ===== Restarting services
 
-Finally, restart the following services:
+Restart the following services:
```

Comment:
You won't be restarting any services

Reply:
Do you propose this entire section be removed?

Reply:
Yes